analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -262,6 +262,13 @@ private fun String.isExternalDocumentReferenceId(): Boolean = startsWith(SpdxCon
  */
 private fun String.mapNotPresentToEmpty(): String = takeUnless { SpdxConstants.isNotPresent(it) }.orEmpty()
 
+/**
+ * Sanitize a string for use as an [Identifier] property where colons are not supported. While spaces theoretically are
+ * supported, replace them with dashes, too, for readability.
+ */
+private fun String.sanitize(): String =
+    replace(Regex("[: ]"), "-").replace(Regex("-{2,}"), "-")
```

Comment:
While I agree that replacing spaces with dashes is a good idea, it will break a lot of curations and I'm not sure if readability is worth it.

Reply:
I don't believe there are many `SpdxDocumentFile`-type curations for packages with spaces in their names around, but I agree we should be conservative still, and I've changed the implementation.